doc/api/tls.md

```diff
@@ -725,6 +725,15 @@ added: v8.4.0
 Disables TLS renegotiation for this `TLSSocket` instance. Once called, attempts
 to renegotiate will trigger an `'error'` event on the `TLSSocket`.
 
+### tlsSocket.enableTrace()
+<!-- YAML
+added: REPLACEME
+-->
+
+When enabled, TLS packet trace information is written to `stderr`. This can be
+used to debug TLS connection problems. The format of the output is undocumented,
+can change without notice, and should not be relied on.
```

Comment:
Maybe mention that the trace output comes from openssl and therefore isn't under our control?

Reply:
Sure, I'll add that. I wanted to add a link to their docs, but they don't doc SSL_trace.